sass/themes/donate/sr18/components/form/_form.scss

```diff
@@ -128,12 +137,18 @@ div#paypal-button-container {
       }
     }
   }
```

Comment:
this giftaid row bit might accidentally be left in when fixing conflicts? Could you have a look there, as I think this is accidentally wrapping another giftaid row and the card logos (which are not appearing now).